src/fparser/two/utils.py

```diff
@@ -418,6 +418,11 @@ def __new__(cls, string, parent_cls=None):
         # Get the class' match method if it has one
         match = getattr(cls, "match", None)
 
+        if _deepcopy:
+            # If this is deep copied (and string is None), simply call
```

Comment:
"is part of a deep-copy operation (and string..." if that's right?

Reply:
Yes.